src/ui/pages/SendTransaction/SendTransaction.tsx

```diff
@@ -277,8 +278,19 @@ function TransactionDefaultView({
           actionTransfers={actionTransfers}
           singleAsset={singleAsset}
           allowanceQuantityBase={allowanceQuantityBase}
-          allowanceViewHref={allowanceViewHref}
         />
+        {allowanceQuantityBase ? (
```

Comment:
I think we also used to check that `action.type` is `approve` before showing this

Reply:
Good point, thanks! Added an extra check for `action.type`

Reply:
✅